azure-pipelines.yml

```diff
@@ -62,7 +62,7 @@ jobs:
         tox.env: 'linting,docs,doctesting'
       py37-xdist:
         python.version: '3.7'
-        tox.env: 'py37-xdist'
+        tox.env: 'py37-lsof-xdist'
```

Comment:
`lsof` tests are unix only, and are skipped on Windows:

```
SKIPPED [2] D:\a\1\s\testing\test_capture.py:952: could not run 'lsof'
```

(From https://pytest-dev.visualstudio.com/pytest/_build/results?buildId=193)

Reply:
Thanks for catching this, fixed.  Moved to Travis.